x-pack/plugins/observability_solution/observability_ai_assistant/server/utils/recall/score_suggestions.ts

```diff
@@ -76,9 +70,9 @@ export async function scoreSuggestions({
 
     Documents:
     ${JSON.stringify(
-      suggestionsWithShortId.map((suggestion) => ({
-        id: suggestion.shortId,
-        content: suggestion.text,
+      suggestions.map((suggestion) => ({
+        ...omit(suggestion, 'score'), // Omit score to not bias the LLM
```

Comment:
Why do we even have `score` here in the first place? We return it from ES but could simply omit it there. Afaict we don't use it. Am I missing anything?

Reply:
Btw. I am now sending the full `suggestion` (except score). Should we still limit to just `id` and `text`? My thinking is that `title` can also be very helpful. `doc_id` is needed in order for the LLM to recognise existing learnings in order to update them rather than creating new entries. Related issue: https://github.com/elastic/kibana/issues/191085